### PR TITLE
NurseConnect Optout submission

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr please review.  not so sure what `optout_type` should be set to when doing identity optout...

**Comment by NicolaasDieBaas** [comment]
review with the knowledge that there are a few optout tests that were skipped that I need to go through and activate if still required... but it seems like `state_optout` might have to change and the tests removed.  Can an user have an active subscription but be opted out..?

**Comment by imsickofmaps** [comment]
👍

File: go-app-ussd_nurse.js [hunk 1]
```diff
@@ -738,8 +737,35 @@ go.app = function() {
...
+                            "action": "nurse_optout",
+                            "data": {
+                                "reason": choice.value
+                            }
+                        };
+
+                        return Q
+                        .all([
+                            // optout identity
+                            is.optout(optout_info),
```
Comment (imsickofmaps) [inline go-app-ussd_nurse.js]: Drop this, it's unsubscribe.

**Comment by imsickofmaps** [comment]
@gsvr remove optout then 👍  for when Travis checked

**Pull request merged**